wayland-scanner/tests/scanner_assets/test-headerless-protocol.xml

```diff
@@ -0,0 +1,252 @@
+<protocol name="test-protocol">
```

Comment:
Given this test file is only used to ensure the scanner doesn't choke on it, I don't think it needs to be as big as the main one, it can be much more minimalistic I'd say.

Reply:
It's done, thanks.